tests/conftest.py

```diff
@@ -191,6 +192,12 @@ def protect_hybrid_cloud_deletions(request):
             seen_models.add(fk_model)
             restrict_role(role="postgres_unprivileged", model=fk_model, revocation_type="DELETE")
 
+    # Protect organization members from being updated without also invoking the correct outbox logic.
+    # If you hit test failures as  aresult of lacking these privileges, first ensure that you create the correct
```

Comment:
```suggestion
    # If you hit test failures as a result of lacking these privileges, first ensure that you create the correct
```

small typo, also the fact that you can restrict this is very cool